modules/i18n/src/translations/fr-FR/portals/console.ts

```diff
@@ -7176,7 +7176,7 @@ export const console: ConsoleNS = {
                 title: "Rôles"
             },
             rolesEdit: {
-                backButton: "Revenir aux {{type}}",
+                backButton: "Revenir aux Groupes",
```

Comment:
Keep the placeholder and change the passing value for the placeholder (to title case) in the places where this message key is used.

Reply:
Changes made accordingly.